### DISCUSSION TITLE
How to use debug mode ?

### DISCUSSION THREAD

**Discussion by LukaszTS901** [diasurgical/DevilutionX#6735] [Q&A] [answered]
Hi !
Can someone explain me how to play Devilutionx in debug mode ? I heard that you need to compile debug build, but I'm not a programmer and I don't know basics of it. Is it any other way to do it? I want to use console commands to experiment with different builds & items in single player just to have fun. I've tried to find character editors or trainers for Devilutionx but nothing seems available at the moment. Thank you in advance for your help.

**Comment by AJenbo** [comment, marked as answer]
Debug commands are for debugging/development, they are not available elsewhere or intended for general use.
You can import saves from the original game, there are plenty of such tools for the original game.
